NEWS.md

```diff
@@ -1,15 +1,16 @@
-# SKELETON
+# hySpc.read.spe 0.0.x 2021-xx-xx
 
 ## Acknowledgements
-The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during summer 2020.
-Erick and the team carried out a significant overhaul of `hyperSpec` which led to this release.
+The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during Summer 2020 and Sang Truong during summer 2021.
```

Comment:
Did Erick work on this package last year?

Reply:
I'm not sure. @eoduniyi Did you work on this last summer? I mention @bryanhanson here too since he might know.

Reply:
Hello! I did not work on this package 🙂:
![Screenshot_20210620-162326.jpg](https://user-images.githubusercontent.com/31193655/122688821-1af5d800-d1e4-11eb-8f52-eb5dc01ba4b8.jpg)

Reply:
Thank you for letting me know!